logstash-integration-jdbc.gemspec

```
@@ -29,7 +29,9 @@ Gem::Specification.new do |s|
   # Restrict use of this plugin to versions of Logstash where support for integration plugins is present.
   s.add_runtime_dependency "logstash-core", ">= 6.5.0"
   s.add_runtime_dependency 'logstash-codec-plain'
-  s.add_runtime_dependency 'sequel', '>= 5.73.0'
+  # Pinned sequel version to >= 5.75.0 as it fixes the generic jdbc adapter to properly
+  # handle disconnection errors, and avoid stale connections in the pool.
+  s.add_runtime_dependency 'sequel', '>= 5.75.0'
```

```suggestion
  # Pinned sequel version to >= 5.74.0 as it fixes the generic jdbc adapter to properly
  # handle disconnection errors, and avoid stale connections in the pool.
  s.add_runtime_dependency 'sequel', '>= 5.74.0'
```

Given the fix was introduced with https://github.com/jeremyevans/sequel/commit/813db795a77687b9a639fe889b038e7a0e46829e and shipped with 5.74.0, maybe we can change this to 5.74.0 instead?

---

Agreed, it makes sense! Thanks!